{
    "config": {
        "abort": {
            "already_configured": "El servicio ya est\u00e1 configurado",
            "reauth_successful": "La autenticaci\u00f3n se volvi\u00f3 a realizar correctamente"
        },
        "error": {
            "cannot_connect": "No se pudo conectar",
            "invalid_auth": "Autenticaci\u00f3n no v\u00e1lida"
        },
        "step": {
            "api_key": {
                "data": {
                    "api_key": "Clave API"
                }
            },
            "reauth_confirm": {
                "data": {
                    "api_key": "Clave API"
                },
                "description": "Por favor, introduce una nueva clave API para PI-Hole en {host}/{location}",
                "title": "Volver a autenticar la integraci\u00f3n Pi-Hole"
            },
            "user": {
                "data": {
                    "api_key": "Clave API",
                    "host": "Host",
                    "location": "Ubicaci\u00f3n",
                    "name": "Nombre",
                    "port": "Puerto",
                    "ssl": "Utiliza un certificado SSL",
                    "statistics_only": "S\u00f3lo las estad\u00edsticas",
                    "verify_ssl": "Verificar el certificado SSL"
                }
            }
        }
    },
    "issues": {
        "deprecated_yaml": {
            "description": "Se va a eliminar la configuraci\u00f3n de PI-Hole mediante YAML. \n\nTu configuraci\u00f3n YAML existente se ha importado a la IU autom\u00e1ticamente. \n\nElimina la configuraci\u00f3n YAML de PI-Hole de tu archivo configuration.yaml y reinicia Home Assistant para solucionar este problema.",
            "title": "Se va a eliminar la configuraci\u00f3n YAML de PI-Hole"
        }
    }
}